### DISCUSSION TITLE
Latest version on helm

### DISCUSSION THREAD

**Discussion by jfsmith-at-coveo** [chaos-mesh/chaos-mesh#3823] [Q&A] [answered]
Hi all!

I'm unfamiliar with how helm works -- do I have to update my sources with a command or something? I would like to upgrade but `helm search repo chaos-mesh` only returns me the following:

```
NAME                 	CHART VERSION	APP VERSION	DESCRIPTION                                       
chaos-mesh/chaos-mesh	2.4.1        	2.4.1      	Chaos Mesh is a cloud-native Chaos Engineering ...

```

This is a bit behind. Is there something I'm missing?

**Comment by fewdan** [comment, marked as answer]
@jfsmith-at-coveo 
You can refer to this [document](https://chaos-mesh.org/docs/production-installation-using-helm/).
Execute the following command, and you will see:
`helm repo add chaos-mesh https://charts.chaos-mesh.org`
If you added it before, you can try to update:
`helm repo update chaos-mesh`
You may see:
![image](https://user-images.githubusercontent.com/16437100/205542474-c7a480d3-8fe7-44eb-8024-22f333ec54c3.png)
And then
`helm search repo chaos-mesh`
Result:
![image](https://user-images.githubusercontent.com/16437100/205541986-d26d0e5a-1bf4-42e1-8798-5c81f2e454ee.png)
And `helm search repo chaos-mesh` will show the latest release of chaos-mesh. 
If you want to install a historical version, execute the following command to view all released versions:
`helm search repo chaos-mesh -l`
You will see:
![image](https://user-images.githubusercontent.com/16437100/205542047-94f024e5-39a2-4c67-a88e-9b66cd88a3a4.png)

**Reply by jfsmith-at-coveo** [reply]
I missed the `update` command. Thank you!
